docs/configuration.md

```diff
@@ -0,0 +1,316 @@
+```
+
+## NodeHealthCheck Status
+
+The status section of the NodeHealthCheck custom resource provides detailed
+information about what the operator is doing. It contains these fields:
+
+| Field                  | Description                                                                                                                                                                                                                                                |
+|------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| _observedNodes_        | The number of nodes observed according to the selector.                                                                                                                                                                                                    |
+| _healthyNodes_         | The number of observed healthy nodes.                                                                                                                                                                                                                      |
+| _inFlightRemediations_ | ** DEPRECATED ** A list of "timestamp - node name" pairs of ongoing remediations. Replaced by unhealthyNodes.                                                                                                                                              |
+| _unhealthyNodes_       | A list of unhealthy nodes and their remediations. See details below.                                                                                                                                                                                       |
+| _conditions_           | A list of conditions representing NHC's current state. Currently the only used type is "Disabled", and it is true when the comtroller detects problems which prevent it to work correctly. See the [workflow page](./workflow.md) for further information. |
+| _phase_                | A short human readable representation of NHC's current state. Known phases are Disabled, Paused, Remediating and Enabled.                                                                                                                                  |
+| _reason_               | A longer human readable explanation of the phase.                                                                                                                                                                                                          |
+
+### UnhealthyNodes
+
+TODO add when documenting escalating remediation
+
+## Remediation Resources
+
+There are two kind of remediation resources involved:
+
+- at least one remediation template CR, which needs to exist when configuring
```

Comment:
Nit: it might be obvious but IMO still worth noting: the connection between template and remediation CR.